src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/InterchangeHeader.java

```diff
@@ -34,26 +41,29 @@ public String getKey() {
     public String getValue() {
         String timestamp = DATE_FORMAT.format(translationTime);
         String formattedSequenceNumber = String.format("%08d", sequenceNumber);
-        return "UNOA:2"+"+"+sender+"+"+recipient+"+"+timestamp+"+"+formattedSequenceNumber;
```

Comment:
From the examples we've gotten through recently it looks like registration messages always end with `++FHSREG'` and RECEP messages always end with `++RECEP+++FHSA EDI TRANSFERS'`

Reply:
new segment: RecepHeader created.